### ISSUE TITLE
Bug: order of face joint idx

### ISSUE THREAD

**Issue by hdjsjyl** [EricGuo5513/HumanML3D#112]
In skeleton.py line58:
      face_joint_idx = [2,1,17,16]
      l_hip, r_hip, sdr_r, sdr_l = face_joint_idx

needs to be changed to :
      r_hip, l_hip, sdr_r, sdr_l = face_joint_idx

Otherwise: the computed forward direction may be z+ or maybe z-.

**Comment by AzureRossi** [comment]
Yes, I mentioned the related bug in issue ：Raw code bug in inverse_kinematics_np Function. #107   as well. However, the author may consider it a minor issue, and anyone who has reviewed the code would likely make a quick fix.
